src/Components/Links.css

```diff
@@ -36,13 +36,4 @@
   white-space: nowrap;
 }
 
-.w-70 {
-  width: 70%;
-  max-width: 45rem;
-}
 
```

Comment:
```suggestion
```

Reply:
i didnt understand exactly what i have to change ?